src/components/vue-form-builder.vue

```diff
@@ -1352,7 +1353,9 @@ export default {
       this.updateState();
       this.inspect(clone);
     },
-
+    orderPages(items) {
+      this.config = [...items];
```

Comment:
Change the order of the pages directly to the screen config is a risk, need to recalculate the references in record lists and navigation buttons, I think the order property for each page is enough.

Add the `this.updateState();` to save the changes in the state (it also needed for undo/redo)